changelog/v1.13.0-beta26/tunneling-upstream-tls.yaml

```
@@ -0,0 +1,7 @@
+changelog:
+- type: FIX
+  issueLink: https://github.com/solo-io/gloo/issues/7313
+  resolvesIssue: true
+  description: |
```

super nit, this formatting ends up being awkward in our rendered changelogs since it persisted newlines (that may not wrap the same in our release notes)
```suggestion
  description: >-
```

---

for more, see https://yaml-multiline.info/

---

479986ff05edaca378aa96ce571d08bbdd32eb14